### PR TITLE
Bump kubevirt 0.37.1

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
This PR bumps kubevirt to 0.37.1 and updates code and manifests to kubevirt/v1

**Special notes for your reviewer**:
PR Solves  #264 

**Release note**:

```release-note
bump kubevirt apiVersion to v1
```

### CODE REVIEW

File: pkg/manager/manager.go [hunk 1]
```diff
@@ -92,6 +92,7 @@ func (k *KubeMacPoolManager) Run(rangeStart, rangeEnd net.HardwareAddr) error {
 			return errors.Wrap(err, "unable to set up manager")
 		}
 
+		os.Setenv("KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION", "v1")
```
Comment (davidvossel) [inline pkg/manager/manager.go:95]: you'll need to set this environment variable before executing the binary. So set it on the pod's container spec, or using an entry point shell script. 

setting it within the binary is too late as the init function that interprets the env var will have already executed.

Comment (RamLavi) [reply pkg/manager/manager.go]: I see, thanks!

File: config/test/kubemacpool.yaml [hunk 2]
```diff
@@ -71,7 +71,7 @@ webhooks:
   - apiGroups:
     - kubevirt.io
     apiVersions:
-    - v1alpha3
+    - v1
```
Comment (qinqon) [inline config/test/kubemacpool.yaml:74]: This add a hard dep on kubevirt >= 0.37 maybe we should document it ?

**Review by qinqon — CHANGES_REQUESTED** [review]

**Comment by qinqon** [comment]
@RamLavi can you fix DCO ? 

/lgtm
/approve

**Comment by RamLavi** [comment]
> @RamLavi can you fix DCO ?

fixed. can you re-lgtm?

**Pull request merged**